validation.js

```diff
@@ -92,8 +92,23 @@ class DatUrlVersion extends LogicalType {
   }
 }
 
+class DateType extends LogicalType {
+  _fromValue (val) {
+    return new Date(val)
+  }
+  _toValue (date) {
+    return date instanceof Date ? +date : undefined
```

Comment:
more explicit:
```suggestion
    return date instanceof Date ? String(date) : undefined
```
or
```suggestion
    return date instanceof Date ? date.toString() : undefined
```